pyproject.toml

```diff
@@ -27,15 +27,15 @@ include = [
 ]
 
 [tool.poetry.dependencies]
-python = ">=3.8,<3.12"
+python = ">=3.8,<=3.12"
```

Comment:
We need this added in the `classifiers` section above too. Take a look at https://github.com/nautobot/nautobot-app-dev-example/pull/36/files for the changes we plan to make for 3.12 support in the cookie

Reply:
I've gone ahead and just copied everything from that update to this PR so Drift Manager has less to do.